letsdrill.rb

```diff
@@ -1,12 +1,28 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+ 	if integer > 89
+ 		return "A"
+ 	elsif integer > 79
+ 		return "B"
+ 	elsif integer > 69
+ 		return "C"
+ 	elsif integer > 59
+ 		return "D"
+ 	end
+ 	return "F"
```

Comment:
Rubyists prefer implicitly returning from methods over calling `return`. This method works the same if you move `"F"` up into an `else` clause, and remove all the `return`s.

https://github.com/bbatsov/ruby-style-guide#no-explicit-return